src/RTCRtpSender.ts

```diff
@@ -1,50 +1,60 @@
-import { NativeModules } from 'react-native';
-
-import MediaStreamTrack from './MediaStreamTrack';
-import RTCRtpCapabilities, { senderCapabilities, DEFAULT_AUDIO_CAPABILITIES } from './RTCRtpCapabilities';
-import RTCRtpSendParameters, { RTCRtpSendParametersInit } from './RTCRtpSendParameters';
+import { NativeModules } from "react-native";
```

Comment:
Please don't reformat the code to match your styling. Make your code conform to the surroundings.